backend/src/main/scala/cromwell/backend/standard/StandardAsyncExecutionActor.scala

```diff
@@ -356,7 +356,7 @@ trait StandardAsyncExecutionActor extends AsyncBackendJobExecutionActor with Sta
         s"""(
            |# add a .file in every empty directory to facilitate directory delocalization on the cloud
            |cd $cwd
-           |find . -type d -empty -print0 | xargs -0 -I % touch %/.file
+           |find . -type d -exec sh -c '[ -z "$$(ls -A "{}")" ] && touch "{}"/.file' \\;
```

Comment:
I fixed this and added this test, but it fails on the workflowsuccess test. Running this workflow locally doesn't give any problems, but on travis it apperently gives a `SYSTEM_ERROR`. I'm not sure what might be causing this, anyone got any ideas?

Reply:
No ideas offhand, I'll try to look at this when I get some time.

Reply:
Running under my own user this fails on TES, PAPI v1 and PAPI v2 due to delocalization issues (the PAPI builds don't run for external contributors for auth reasons). Khalid pointed out that the line `File empty_dir = dir_name` that treats a directory as a file may be nonportable WDL that causes problems with platforms that try to delocalize outputs. While a test like this would be nice, I don't want to hold up this PR any longer so I'd just remove the test for now.

Unrelated, I would suggest a minor tweak to the single quote escaping as [described here](https://stackoverflow.com/a/1250279). Basically this double quotes only the single quotes which should eliminate the danger of unintended interpolation.

```find . -type d -exec sh -c '[ -z "$$(ls -A '"'"'{}'"'"')" ] && touch '"'"'{}'"'"'/.file' \\;```

Reply:
@mcovarr Thanks for looking into this! I made the changes you suggested.